CHANGES.md

```diff
@@ -10,6 +10,27 @@
 
 [//]: # (towncrier release notes start)
 
+## 0.3.0 (2024-10-02) {: #0.3.0 }
+
+
+
+#### Features {: #0.3.0-feature }
+
+- Bump dependency on pulp-cli up to version 0.29.*.
```

Comment:
Honestly, such a bump only, without any other code changes, could just go into a z-stream release.
(That probably does not help the build team a lot though.)

Reply:
I was wondering if that was an option as well, but ultimately just did a Y release as the path of least resistance. I fell like until we decide to bump the version to `1.0.0` it is kind of "fine" to just do whatever works for now.